pkgs/top-level/php-packages.nix

```diff
@@ -478,50 +513,6 @@ let
     };
   };
 
-  pinba = if isPhp73 then pinba73 else pinba7;
-
-  pinba7 = assert !isPhp73; buildPecl {
```

Comment:
Are we still using this approach somewhere to ensure that an extension is only built against supported PHP versions? If yes, we can probably do some sort of `disabled = <cond>` (as it's done in python packages) IMHO.

Reply:
I got rid of all uses in `php-packages.nix` by replacing them with `meta.broken` or, in the case of `pinba` and `pthreads`, removing them. A `disabled` param would be nice, though :)